src/cswinrt/code_writers.h

```diff
@@ -10230,19 +10230,16 @@ return true;
             return;
         }
 
-        if (is_flags_enum(type))
-        {
-            w.write("[FlagsAttribute]\n");
-        }
-
         auto enum_underlying_type = is_flags_enum(type) ? "uint" : "int";
 
-        w.write(R"(%%%% enum % : %
+        w.write(
+R"(
+%[WindowsRuntimeMetadata("Windows.Foundation.FoundationContract")]
```

Comment:
Not always foundation contract, will need to call that metadata function to write it.